### PR TITLE
Missing tokens

### PR DESCRIPTION
**Pull request by djaciel**
- add version released to npm
- add missing tokens from namedAccounts

### CODE REVIEW

File: src/tokens/ethereum.ts [hunk 1]
```diff
@@ -42,4 +42,36 @@ export const tokens: IToken[] = [
...
+    chainId: ChainId.Ethereum,
+    enabled: true,
+    decimals: 18,
+    symbol: 'BUSD',
+  },
+  {
+    name: 'SHIB',
+    address: '0x95aD61b0a150d79219dCF64E1E6Cc01f0B64C4cE',
+    chainId: ChainId.Ethereum,
+    enabled: true,
```
Comment (pmprete) [inline src/tokens/ethereum.ts:57]: This should be disabled

File: src/tokens/ethereum.ts [hunk 2]
```diff
@@ -42,4 +42,36 @@ export const tokens: IToken[] = [
     decimals: 18,
     symbol: 'ETH',
   },
+  {
+    name: 'BUSD',
+    address: '0x4Fabb145d64652a948d72533023f6E7A623C7C53',
+    chainId: ChainId.Ethereum,
```
Comment (pmprete) [inline src/tokens/ethereum.ts:48]: this should be disabled

**Review by pmprete — CHANGES_REQUESTED** [review]

File: src/tokens/bsc.ts [hunk 3]
```diff
@@ -90,4 +90,12 @@ export const tokens: IToken[] = [
     decimals: 18,
     symbol: 'BNB',
   },
+  {
+    name: 'BETH',
+    address: '0x250632378E573c6Be1AC2f97Fcdf00515d0Aa91B',
+    chainId: ChainId.BSC,
+    enabled: false,
```
Comment (AWolfsdorf) [inline src/tokens/bsc.ts:97]: Is ok that this token is disabled?

**Review by AWolfsdorf — APPROVED** [review]
Nice work! I just added a question about if the added token should be disabled, if it was intended just ignore my comment

**Review by LizanLycan — APPROVED** [review]
LGTM

**Pull request closed without merging**